src/config.cr

```diff
@@ -107,4 +107,7 @@ module Shards
   class_property? local = false
   class_property? skip_postinstall = false
   class_property? skip_executables = false
+
+  # Defaults to 1 (disabled) when running in `crystal spec`
+  class_property parallel_fetch : Int32 = {{ @top_level.has_constant?("Spec") ? 1 : 8 }}
```

Comment:
The idea was to avoid potential hangs / undefined behaviour when tests run in parallel.

But your comment below made me realize that was misguided for a whole
bunch of reasons (no least: afaik Spec doesn't even have concurrency [[yet?](https://github.com/crystal-lang/crystal/issues/8289)]). 

**tdlr:** I've removed that special behaviour, it now defaults to 8 unconditionally.